config/examples/Creality/Ender-5 Pro/BigTreeTech SKR Mini E3 3.0 with BLTouch-HighTemp/ReadMe.md

```diff
@@ -0,0 +1,15 @@
+Requires the following chage to "...\Marlin\src\module\temperature.cpp"
+
+Comment the sanity check for temperature error as following 
```

Comment:
Don't do this. You're exceeding the thermistor table's max temperature, which is why we added this sanity check.

Reply:
Should I edit the table instead?

The thermistor I installed is rated for 320.

Reply:
> Should I edit the table instead?

No. You should select the correct thermistor.

Reply:
Ok. I will find the right one. Thank you